src/Elastic.Apm/Model/Transaction.cs

```diff
@@ -25,12 +25,12 @@ internal class Transaction : ITransaction
 		private readonly IApmServerInfo _apmServerInfo;
 		private readonly Lazy<Context> _context = new Lazy<Context>();
 		private readonly ICurrentExecutionSegmentsContainer _currentExecutionSegmentsContainer;
-
 		private readonly IApmLogger _logger;
 		private readonly IPayloadSender _sender;
-
 		private readonly string _traceState;
 
+		private bool _createdActivity;
```

Comment:
Could we add a short comment on this one? Something like:
```
Flags whether the agent fired a new `Activity` for this transaction or not.
If there is an already existing transaction, no new transaction will be created and this bool remains false.
```

I'm saying this because [this part in the `ctor`](https://github.com/elastic/apm-agent-dotnet/pull/1228/files#diff-16bfd2f9d441e32ed34c366af81982b949e3857e24ef2abc53e89384b1538344R123-R131) took me some time to understand. With such comment hopefully it'll be easier to understand that part of the code.

Reply:
done 👍